src/WebJobs.Script.WebHost/Diagnostics/KubernetesEventGenerator.cs

```diff
@@ -33,12 +33,12 @@ public override void LogFunctionTraceEvent(LogLevel level, string subscriptionId
                 traceLog.Add("FunctionName", functionName);
                 traceLog.Add("EventName", eventName);
                 traceLog.Add("Source", source);
-                traceLog.Add("Details", NormalizeString(details));
-                traceLog.Add("Summary", NormalizeString(summary));
+                traceLog.Add("Details", NormalizeString(details).Trim('"'));
```

Comment:
NormalizeString adds the surrounding double quotes, then you're trimming them. Instead, I think it'd be cleaner to change the signature of the method to `NormalizeString(string value, bool addEnclosingQuotes = true)`. Then in your case you just call `NormalizeString(details, addEnclosingQuotes = false)`.